db/schema.rb

```diff
@@ -124,16 +113,19 @@
     t.string   "city"
     t.string   "state"
     t.string   "zip_code"
-    t.datetime "created_at",               null: false
-    t.datetime "updated_at",               null: false
+    t.datetime "created_at",                               null: false
+    t.datetime "updated_at",                               null: false
     t.integer  "age"
     t.string   "gender"
     t.string   "race"
     t.string   "size"
     t.string   "wish_list"
     t.integer  "recipient_family_id"
+    t.integer  "membership_id"
```

Comment:
that's how I added the column in the [migration file](https://github.com/rubyforgood/share_christmas/blob/9b17bd4ba0fad009382c2e653390253f32a6df8e/db/migrate/20160621034132_remove_match.rb#L3)